docs/extensions/reconcile-trigger.md

```diff
@@ -0,0 +1,13 @@
+# Reconcile trigger
+
+Gardener dictates the time of reconciliation for resources of the API group `extensions.gardener.cloud`.
+It does that by annotating the respected resource with `gardener.cloud/operation=reconcile`.
+Extension controllers shall react to this annotation and start reconciling the resource.
+They have to remove this annotation as soon as they begin with their reconcile operation and maintain the `status` of the extension resource accordingly.
+
+The reason for this behaviour is that it is possible to configure Gardener to reconcile only in the shoots' maintenance time windows.
+In order to avoid that the extension controller reconcile outside of the shoot's maintenance time window we have introduced this contract.
```

Comment:
In order to avoid that extension controllers reconcile outside of the shoot's maintenance time window we have introduced this contract.